### DISCUSSION TITLE
how to add buildpacks?

### DISCUSSION THREAD

**Discussion by AliAryanTech** [vercel/community#1258] [Help]
### Summary

Like ffmpeg libwebp imagemagick buildpacks?

### Additional information

Skip

### Example

Skip

**Comment by amyegan** [comment]
Hi @AliAryanTech! Can you explain a little more about what you're trying to do? These tools seem to be made for more traditional server environments, but, depending on what you want to accomplish, serverless alternatives may be available.

**Reply by AliAryanTech** [reply]
Nix buildpacks supported?
